projects/fsastorefrontlib/src/shared/util/helpers/date.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { FormatDatePipe } from './formatDate.pipe';
+import { ParseDatePipe } from './parseDate.pipe';
+
+@NgModule({
+  declarations: [FormatDatePipe, ParseDatePipe],
+  exports: [FormatDatePipe, ParseDatePipe],
+})
+export class FsDateModule {}
```

Comment:
No need for FS prefix.
See https://wiki.msg-global.com/display/FSA/02+-+Naming+convetions for more info

Reply:
`FormatDateModule` since pipe is `cxFormatDate`?

Reply:
Module doesn't include only cxFormatDate pipe, so I am not sure if there is need to rename to FormatDateModule or to stay  as it is DateModule. I had an idea to wrap all date pipes in one module to use them on multiple places.

Reply:
Then maybe `DateFormatConfigurationModule` since both pipes are related to some date formatting configuration? `DateModule` sounds a bit to generic for this purpose in my opinion...